api/authFunctions.ts

```diff
@@ -0,0 +1,25 @@
+import {account } from "./appwrite"
+import { useRouter } from 'next/navigation';
+
+   export const logout = async () => {
```

Comment:
Did we decide on Functional or Class functions? @jasonetorres' has Class functions while this one is Functional.

Reply:
We have decided to do functional - easier to export out and import into other files. Please let me know if you have any other thoughts.